### PR TITLE
Re-adds the Red Winter Coat

### PR DESCRIPTION
**Pull request by furrycactus**
Re-adds the former redsec winter coat as a generic red winter coat. It's the same coat from the [picture on the Empire of Dominia wiki page](https://wiki.aurorastation.org/images/3/3b/Dom410x320.png).

Doesn't count as standard security uniform, and doesn't carry any armour values like the security winter coat whatsoever, either. But I guess Necropolis contractors could get away with it.

### CODE REVIEW

**Comment by furrycactus** [comment]
Why did it immediately request skull aaaaaaaa

File: html/changelogs/furrycactus - red winter coat.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: Furrycactys
```
Comment (Geevies) [inline html/changelogs/furrycactus - red winter coat.yml]:
```suggestion
author: Furrycactus
```

Comment (furrycactus) [reply html/changelogs/furrycactus - red winter coat.yml]: LMFAO thanks Geeves

**Comment by furrycactus** [comment]
!review

**Comment by furrycactus** [comment]
Actually don't review it yet, I'm gonna condense all the winter coats into a selection instead.

**Comment by furrycactus** [comment]
Nvm I can't because several of them are tied to job restrictions.

**Comment by furrycactus** [comment]
!review

**Pull request merged**